tests/test_pipenv.py

```diff
@@ -602,12 +626,12 @@ def test_specific_package_environment_markers(self):
     @pytest.mark.tablib
     def test_install_editable_git_tag(self):
         with PipenvInstance() as p:
-            c = p.pipenv('install -e git+git://github.com/kennethreitz/tablib.git@v0.12.1#egg=tablib')
+            c = p.pipenv('install -e git+https://github.com/kennethreitz/tablib.git@v0.12.1#egg=tablib')
```

Comment:
Why are we changing this? If the user types `git+git://` we should still be handling it the same.

Reply:
Apparently `git+git://` is a deprecated protocol